dask/compatibility.py

```diff
@@ -265,3 +265,20 @@ class to receive bound method
         setattr(cls, name, types.MethodType(func, None, cls))
     else:
         setattr(cls, name, func)
+
+
+# Borrowed from six
```

Comment:
Do we need to include a license?

Reply:
Depends on your reading of substantial portions. 😜 Though probably best to be safe and include it.

Reply:
Removed this code completely to something simpler, no need anymore.